### PR TITLE
Feature/card element content

### PR DESCRIPTION
**Pull request by fabien0102**
### Related issue

#478

### Impacted component

https://deploy-preview-493--operational-ui.netlify.com/docs/#card

### Notes

* it's just the inside of the card, we need to add more example when #490 will be merge to have the full card design :wink:
* I fixed the typography documentation (styleguidist auto import only if we have only one component export by file)

### CODE REVIEW

**Comment by fabien0102** [comment]
[Card content elements](https://trello.com/c/MEc13ROP/175-card-content-elements)

**Comment by micha-f** [comment]
Only based on the looks - not code - great!

**Comment by TejasQ** [comment]
About CardColumns, perhaps CSS Grid is a better tool for aligning the columns. This gives us proper column stacking out of the box [according to this sample](https://codepen.io/Weldebob/pen/WjMLav).

**Comment by micha-f** [comment]
👍 grid-gap is a wonderful help for these columns

File: packages/components/src/Card/README.md [hunk 1]
```diff
@@ -5,11 +5,108 @@ Cards are used to group and lay out content on the interface - in fact, non-scro
...
+  updated: "tomorrow",
+  repo: "https://git.com/hello.git",
+}
+;<Card title="Details" data={myData} keyFormatter={title => `-- ${title} --`} />
+```
+
+### With value formatters
+
+```jsx
+const DeployedStatus = ({ deployed }) => (
```
Comment (TejasQ) [inline packages/components/src/Card/README.md]: Status is now in master. Please use this.

File: packages/components/src/CardItem/CardItem.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+export interface Props {
+  /** Title of the item */
+  title: string
+  /** Value of the item */
+  value: React.ReactNode
+}
+
+const CardItemTitle = glamorous.div(
+  ({ theme }: WithTheme): CssStatic => ({
+    color: "#909090",
```
Comment (TejasQ) [inline packages/components/src/CardItem/CardItem.tsx]: Can we use the constant here?

File: packages/components/src/CardItem/CardItem.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    fontFamily: theme.fontFamily,
+    textTransform: "uppercase",
+    fontSize: theme.typography.small.fontSize,
+  }),
+)
+
+const CardItemBody = glamorous.div(
+  ({ theme }: WithTheme): CssStatic => ({
+    ...theme.typography.body,
+    color: "#545454",
```
Comment (TejasQ) [inline packages/components/src/CardItem/CardItem.tsx]: Constant please.

File: packages/components/src/CardColumn/CardColumn.tsx [hunk 4]
```diff
@@ -1,54 +1,34 @@
...
-    fontSize: 14,
-    borderBottom: "1px solid #e8e8e8",
-    marginBottom: theme.spacing,
-  }),
-)
+const Title = styled("div")(({ theme }: { theme?: OperationalStyleConstants }) => ({
+  fontFamily: theme.font.family.main,
+  color: theme.color.text.default,
+  fontSize: 14,
+  borderBottom: "1px solid #e8e8e8",
```
Comment (TejasQ) [inline packages/components/src/CardColumn/CardColumn.tsx]: Use a color from the theme here please.

Comment (TejasQ) [reply packages/components/src/CardColumn/CardColumn.tsx]: `theme.colors.separator.default`

File: packages/components/src/CardColumns/CardColumns.tsx [hunk 5]
```diff
@@ -1,29 +1,13 @@
...
-  <Container id={props.id} css={props.css} className={props.className}>
-    {props.children}
-  </Container>
-)
+import styled from "react-emotion"
+
+const CardColumn = styled("div")(({ children }: React.Props<{}>) => ({
+  display: "flex",
+  flexWrap: "wrap",
+  margin: -11,
```
Comment (TejasQ) [inline packages/components/src/CardColumns/CardColumns.tsx]: Where does this come from?

File: packages/components/src/CardHeader/CardHeader.tsx [hunk 6]
```diff
@@ -1,46 +1,45 @@
...
+  // This ensures that the card header text and card controls are placed in opposite corners.
+  justifyContent: "space-between",
+  height: 40,
+  margin: -20,
+  marginBottom: 20,
+  padding: `0 20px`,
+  lineHeight: 1,
+  "& > :not(:first-child)": {
+    fontSize: 12,
+    color: "#909090",
```
Comment (TejasQ) [inline packages/components/src/CardHeader/CardHeader.tsx]: Color from the theme, please. `theme.colors.text.lightest`.

File: packages/components/src/CardItem/CardItem.tsx (hunk 2 above)
Comment (TejasQ) [reply packages/components/src/CardItem/CardItem.tsx]: `theme.colors.text.lightest`

File: packages/components/src/CardItem/CardItem.tsx (hunk 3 above)
Comment (TejasQ) [reply packages/components/src/CardItem/CardItem.tsx]: `theme.colors.text.default`

**Pull request merged**